blocks/components/editable/index.js

```diff
@@ -44,9 +44,9 @@ export default class Editable extends wp.element.Component {
 	onSetup( editor ) {
 		this.editor = editor;
 		editor.on( 'init', this.onInit );
-		editor.on( 'focusout', this.onChange );
 		editor.on( 'NewBlock', this.onNewBlock );
 		editor.on( 'focusin', this.onFocus );
+		editor.on( 'input', this.onChange );
```

Comment:
Without any content processing, and with the content size being so small, I think this is fine if we keep it in mind. For blocks like freeform, less so.